portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminTaxonomyAPI/ExposeErcForCategories.testcase

```diff
@@ -62,4 +131,68 @@ definition {
 		}
 	}
 
+	@priority = "4"
+	test CanRecieveErrorMessageByNotExistErcInCategory {
+		property portal.acceptance = "true";
+
+		task ("And Given a TaxonomyCategory with custom external reference code") {
+			var taxonomyVocabularyId = TaxonomyVocabularyAPI.getElementOfCreatedVocabulary(
+				element = "$.id",
+				responseToParse = "${responseOfCreated}");
+
+			var responseOfCategory = TaxonomyCategoryAPI.createTaxonomyCategory(
+				externalReferenceCode = "erc",
+				name = "Category Name",
+				taxonomyVocabularyId = "${taxonomyVocabularyId}");
+		}
+
+		task ("When using a GET request getTaxonomyVocabularyTaxonomyCategoryByExternalReferenceCode with not exist erc value") {
+			var response = TaxonomyCategoryAPI.getTaxonomyCategoryByErc(
+				externalReferenceCode = "liferay",
+				taxonomyVocabularyId = "${taxonomyVocabularyId}");
+		}
+
+		task ("Then I can receive the error message in the response") {
+			var siteId = TaxonomyCategoryAPI.getElementOfCreatedCategory(
+				element = "$.siteId",
+				responseToParse = "${responseOfCategory}");
+			var actualValue = JSONUtil.getWithJSONPath("${response}", "$.title");
```

Comment:
I'm a little curious is it necessary to keep both getElementOfCreatedCategory and JSONUtil.getWithJSONPath()? I know we have getElement() or getWithJSONPath in other tests, maybe this is something we can improve in the future if John and Meg would like to.